pages/common/pyinfra.md

```diff
@@ -0,0 +1,28 @@
+# pyinfra
+
+> Automates infrastructure at a large scale.
+> More information: <https://docs.pyinfra.com>.
+
+- Execute a command over ssh:
+
+`pyinfra {{target_ip_address}} exec -- {{command_name_and_arguments}}`
+
+- Execute contents of a deploy file on a list of targets:
+
+`pyinfra {{path/to/inventory.py}} {{path/to/deploy.py}}`
```

Comment:
I'm not sure if it makes sense here, but it's a bit unclear what `inventory.py` is.
```suggestion
`pyinfra {{path/to/target_list.py}} {{path/to/deploy.py}}`
```